### PR TITLE
media-sound/yesplaymusic: A third party music player for Netease Music

### PR DESCRIPTION
**Pull request by zozx**
Signed-off-by: Raymond Kwok <c2778148787@gmail.com>

### CODE REVIEW

File: media-sound/yesplaymusic-bin/yesplaymusic-bin-0.4.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop xdg
+
+DESCRIPTION="A third party music player for Netease Music"
+HOMEPAGE="https://github.com/qier222/YesPlayMusic"
+BASE_URI="https://github.com/qier222/YesPlayMusic/releases/download/v0.4.2"
```
Comment (Universebenzene) [inline media-sound/yesplaymusic-bin/yesplaymusic-bin-0.4.2.ebuild]: `0.4.2` 换成 `${PV}`

File: media-sound/yesplaymusic-bin/yesplaymusic-bin-0.4.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+src_configure(){
+	rm -r "${S}/usr/share/applications" || die
+}
+
+src_install(){
+	insinto "/opt"
+	doins -r "${S}/opt/YesPlayMusic"
+	insinto "/usr/share"
+	doins -r "${S}/usr/share/icons"
```
Comment (Universebenzene) [inline media-sound/yesplaymusic-bin/yesplaymusic-bin-0.4.2.ebuild]:
```bash
for si in 16 24 32 48 64 128 256 512; do
	doicon -s ${si) usr/share/icons/hicolor/${si}x${si}/apps/${PN%-bin}.png
done
```
1024不是经典分辨率，也可以考虑装pixmap去，不加-s选项就行

File: media-sound/yesplaymusic-bin/yesplaymusic-bin-0.4.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	insinto "/usr/share/applications"
```
Comment (Universebenzene) [inline media-sound/yesplaymusic-bin/yesplaymusic-bin-0.4.2.ebuild]:
```bash
domenu "${FILESDIR}/${PN%-bin}.desktop"
```

**Review by Universebenzene — COMMENTED** [review]
好像是这么review？

**Review by Universebenzene — APPROVED** [review]
没啥意见了，感觉行

**Review by RevySR — APPROVED** [review]

**Pull request merged**